src/varpubs/cli.py

```diff
@@ -1,24 +1,61 @@
+'''
+-------------------------------------------------------------
+Usage:
+1) Deploy the database by running:
+
+python -m varpubs.cli deploy-db \
+    --db_path PATH/TO/OUTPUT_DB.duckdb \
+    --vcf_paths PATH/TO/FILE1.vcf PATH/TO/FILE2.vcf ... \
+    --email your_email@example.com
+
+Description:
+- Parses one or more VCF files to extract HGVS.p terms.
+- Queries PubMed using Entrez API and stores abstracts in DuckDB.
+- Requires your email for NCBI Entrez API access.
+
+2) After running, to inspect the database:
+
+Option A - Using DuckDB CLI:
+duckdb PATH/TO/OUTPUT_DB.duckdb
+
+Then inside the CLI, run:
+> SELECT * FROM pubmedarticle;
+
+Option B - Export table(s) to CSV from inside DuckDB:
+> COPY pubmedarticle TO 'pubmed_articles.csv' (HEADER, DELIMITER ',');
+
+-------------------------------------------------------------
+'''
+
 from simple_parsing import ArgumentParser
 from dataclasses import dataclass
 from pathlib import Path
+from typing import List
 from varpubs.pubmed_db import PubmedDB
 
-
 @dataclass
-class Settings:
+class DeployDBArgs:
     db_path: Path
+    vcf_paths: List[Path]
+    email: str
 
+def main():
+    parser = ArgumentParser()
+    subparsers = parser.add_subparsers(dest="command", required=True)
 
-def get_argument_parser() -> ArgumentParser:
-    parser: ArgumentParser = ArgumentParser()
-    parser.add_arguments(Settings, dest="options")
-    subparsers = parser.add_subparsers(help="subcommand help")
-    subparsers.add_parser("deploy-db", help="Deploy the database")
-    return parser
+    deploy_parser = subparsers.add_parser("deploy-db", help="Deploy the database")
+    deploy_parser.add_arguments(DeployDBArgs, dest="args")
 
+    args = parser.parse_args()
 
-def main():
-    args = get_argument_parser().parse_args()
-    db = PubmedDB(path=args.db_path)
-    if args.subparser_name == "deploy-db":
+    if args.command == "deploy-db":
+        db = PubmedDB(
+            path=args.args.db_path,
+            vcf_paths=args.args.vcf_paths,
+            email=args.args.email
+        )
         db.deploy()
+
+if __name__ == "__main__":
+    print(">>> CLI module invoked")
+    main()
```

Comment:
This is still a remainder from calling varpubs via `python` and not `pixi` so it can go.